### DISCUSSION TITLE
I am having issues with verifying my academic affiliation

### DISCUSSION THREAD

**Discussion by SidmoGoesBrrr** [community/community#68909] [GitHub Education] [answered]
### Hi everyone,

I have applied countless times. With my fee receipt, my college ID, etc etc. Every single time, I get rejected with the reason Attendees of this school must verify and use their academic email address.
I have specified this before countless times to GitHub support when they have not responded

**Comment by harxh-v** [comment]
Dear @SidmoGoesBrrr 


**Explanation**
To confirm your eligibility for education benefits on GitHub, it is necessary to use your academic or college email address.

The option to verify your education status using documents like fee receipts and college IDs is intended for students from institutions not directly affiliated with GitHub.

**Solution:**
Kindly contact your school or college for creation of your academics email, as your school or college is already affiliated with GitHub.

**Reply by SidmoGoesBrrr** [reply]
No but that is the confusion. As I said in my previous post, my institution has a junior college and a degree college. Junior college is basically junior and senior year of high school in India, and degree college with bachelors and masters. I am enrolled in the junior college, where we do not get academic emails. Only those is degree college do. The location and website for the college is same.

**Reply by SidmoGoesBrrr** [reply]
No but that is the confusion. As I said in my previous post, my institution has a junior college and a degree college. Junior college is basically junior and senior year of high school in India, and degree college with bachelors and masters. I am enrolled in the junior college, where we do not get academic emails. Only those is degree college do. The location and website for the college is same.

**Reply by harxh-v** [reply]
Please refer to [GitHub Education Terms](https://education.github.com/schools/terms) 
They don't allow school students to join the program, you need to join a high school first.
Kindly, mark the post as answered.
Thank you!

**Reply by SidmoGoesBrrr** [reply]
Okay I'm gonna be as patient and explain this as calmly as I can. Ramnarain Ruia Autonomous College of Arts and Science is a degree college with Bachelors programs, masters programs, PHDs etc. Those who are enrolled in these programmes get the college email address, **which I am not**. I am in Ramnivas Ruia Junior College, which is the last 2 years of highschool. In my city, a student does 10 years of school, passes the board exam, moves on to Junior College where they complete 2 years of school i.e 11th grade and 12th grade and give the 12th grade board exam. I had github education in 11th grade, where I uploaded a photo of my ID. I did the same this year when it expired with my new 12th grade/ SYJC (Second Year Junior College) ID card, fee receipt and screenshot of portal. It still auto rejected, telling me I needed to use an academic email. As per the terms of service, I am in a highschool so I should be eligible for Github Education. To recap, Ramnarain Ruia Autonomous College of Arts and Science, the degree college and Ramnivas Ruia Junior College (Last 2 years of high school) are owned by the same public organisation. However one of them gives academic email addresses and one of them doesn't. I fall in the latter category and hence tried countless number of times to verify by other means.

**Reply by harxh-v** [reply, marked as answer]
Hey Siddhant,

As I stated earlier, [GitHub Education Terms](https://education.github.com/schools/terms) have been updated recently. And now, they aren't approving or onboarding anyone to GitHub education with any verification method, except edu email.

They haven't updated the onboarding process yet. Till then whenever you reapply, your request will be automatically rejected.

Kindly mark the answer as answered.
Thank you!

**Reply by harxh-v** [reply]
Further, 

Ping @iHRex (me) up at [LinkedIn](https://www.linkedin.com/in/ihrex) 

I'd love to share some tricks with you! 😉
Hope you get it!

**Reply by trito2** [reply]
I uploaded my faculty ID along with my teaching schedule, showing my name and term for Fall and Spring 2024. But the app kept ejecting my ID because the school name was not clear on the ID. Is there anyway I can submit my faculty ID, as I don't have anything else to submit now?

**Comment by sreshtech** [comment]


**Reply by SidmoGoesBrrr** [reply]
Hey. I believe you didn't read what I have written in detail. My school does not issue me an email address. Github everytime auto rejects me so there is nothing to wait for. I tried *ALL* the alternative verification methods

**Comment by Dawit0101** [comment]
Here are some steps you can take to address this issue:

1. Double-Check Information: Make sure that the information you are providing during the verification process is accurate and up-to-date. Ensure that you are using your official academic email address and that the documents you provide, such as your college ID and fee receipt, are clear and legible.

2. Use Official Email: If the verification process specifically requires you to use an academic email address, ensure that you are using the official email address provided by your educational institution. Sometimes, personal email addresses may not be accepted for academic verification.

3. Contact Your Institution: Reach out to your academic institution's IT department or administrative office to verify that your official academic email address is set up and functioning correctly. They may be able to assist you with any email-related issues.

4. GitHub Support: Continue to reach out to GitHub support through their official channels, such as their support email or help center. While it can be frustrating when you don't receive a response immediately, persistence may eventually lead to a resolution.

5. Provide Clear Documentation: When submitting verification documents, make sure they are clear and provide all the necessary information. If there is any ambiguity in the documents, it might result in rejection.

6. Patience: Sometimes, the verification process can take time, especially if there is a backlog of requests. Be patient and continue to follow up with GitHub support periodically.

7. Appeal Process: In some cases, there may be an appeal process available if your initial verification request is denied. Check GitHub's documentation or reach out to their support to inquire about the possibility of an appeal.

8. Alternative Options: If all else fails, consider exploring alternative options for accessing GitHub's education-related benefits. For example, some educational institutions have GitHub Campus Programs that provide access to GitHub for students, so you could check if your institution participates in such a program.

**Comment by samus-aran** [comment]
Hi @SidmoGoesBrrr 👋 

Re-reading through your situation there are a couple of options:

1. In your support tickets (if open) you provide proof from the institution that they do not provide academic emails to the attendees of Junior College. This should be appropriate evidence whether a letterhead, or a student handbook. It would be even better if they provided that on their website so that our analysts can confirm.
2. Raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". Include the evidence requested.

We can then pass this along to the internal team to analyses and verify the situation. Hope this helps 👍 

EDIT: I can see the request I have made previously and this (above) is required. 🙂
